src/components/Media/Media.tsx

```diff
@@ -2,16 +2,25 @@ import type { UploadcareImage } from '@prezly/uploadcare';
 import classNames from 'classnames';
 import type { CSSProperties } from 'react';
 
+import { convertSizesToWidths } from './lib';
+
 import './Media.scss';
 
 interface Props {
     className?: string;
     image: UploadcareImage;
     style?: CSSProperties;
     title?: string;
+    sizes?: string;
 }
 
-export function Media({ className, image, style, title }: Props) {
+export function Media({
+    className,
+    image,
+    style,
+    title,
+    sizes = '(max-width: 992px) 800px, (max-width: 576px) 400px, 1200px',
```

Comment:
Is the order correct here? Should it no go from the lowest to the highest max-width values?

```suggestion
    sizes = '(max-width: 576px) 400px, (max-width: 992px) 800px, 1200px',
```

Reply:
Not sure if the order actually matters here, couldn't find anything pointing to that 🤔

Reply:
I've tested this in JSFiddle, and it seems they _have_ to follow the lowest-to-highest order.

Compare these two:

- https://jsfiddle.net/30wyqjzv/
- https://jsfiddle.net/30wyqjzv/1/